src/main/resources/data.sql

```diff
@@ -1,14 +1,18 @@
 SET FOREIGN_KEY_CHECKS = 0;
 
 INSERT INTO grade (code, name, organisation_id) VALUES
-	('AA', 'Administrative level',null),
+	('AA', 'Administrative Assistant',null),
 	('AO', 'Administrative Officer',null),
     ('EO', 'Executive Office',null),
-	('SEO','Senior Executive Office',null),
-    ('HEO','Higher Executive manager',null),
- 	('G6', 'Grade 6', null),
-  	('G7', 'Grade 7', null),
-	('SCS', 'Director', null);
+   	('G6', 'Grade 6', null),
+    ('G7', 'Grade 7', null),
+    ('HEO','Higher Executive manager', null),
```

Comment:
Should this be `Higher Executive Officer` as per the ticket?

Reply:
yup, changed